agent/src/main/java/io/honeycomb/opentelemetry/HoneycombAgent.java

```diff
@@ -48,7 +48,9 @@ private static void configureEnvironment() throws Exception {
         }
 
         System.setProperty("otel.exporter.otlp.headers",
-            String.format("x-honeycomb-team=%s,x-honeycomb-dataset=%s", apiKey, dataset));
+            String.format("%s=%s,%s=%s",
+                EnvironmentConfiguration.HONEYCOMB_TEAM_HEADER, apiKey,
+                EnvironmentConfiguration.HONEYCOMB_DATASET_HEADER, dataset));
```

Comment:
Is case important here? We've gone from lowercase to mixed case.

Reply:
I did verify that it still works with mixed-case. The SDK builder was already using mixed-case, and as far as I can tell, they share the underlying mechanism: https://github.com/open-telemetry/opentelemetry-java/blob/ceb31c6caf272390c2703c81aa33357a24449cc7/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L81

IDK if the Java transport does anything with the casing, or our API doesn't care about it, maybe both.